bin/mkreleaselog

```diff
@@ -109,6 +144,17 @@ pr_link() {
     printf -- "[%s#%s](https://%s/pull/%s)" "$ghname" "$prnum" "$repo" "$prnum"
 }
 
+ignored_commit() {
+    local commit="$1"
+    local matches
+
+    # Check to see if this commit includes any non-ignored files.
+    # Use jq instead of grep so we get PCRE regexes.
```

Comment:
Maybe I'm just being bad at bash, but where's the jq here?

Reply:
Ah, that use of jq was stillborn. Let me fix the comment.